test/systemtests/aci_test.go

```diff
@@ -72,403 +73,582 @@ func (s *systemtestSuite) TestACIMode(c *C) {
 	c.Assert(node1.runCommandUntilNoError("pgrep netplugin"), IsNil)
 	time.Sleep(20 * time.Second)
 
-	// Verify cA1 can ping cA2
 	c.Assert(s.pingTest(containersA), IsNil)
-	// Verify cB1 can ping cB2
 	c.Assert(s.pingTest(containersB), IsNil)
-	// Verify cA1 cannot ping cB1
 	c.Assert(s.pingFailureTest(containersA, containersB), IsNil)
 
 	c.Assert(s.removeContainers(containersA), IsNil)
 	c.Assert(s.removeContainers(containersB), IsNil)
 	c.Assert(s.cli.EndpointGroupDelete("default", "epga"), IsNil)
 	c.Assert(s.cli.EndpointGroupDelete("default", "epgb"), IsNil)
-	c.Assert(s.cli.NetworkDelete("default", "aciNet"), IsNil)
+	c.Assert(s.cli.NetworkDelete("default", s.globInfo.Network), IsNil)
 }
 
-/*
+/* TesACIPingGateway checks ping success from containers running in a EPG to the default gateway */
 func (s *systemtestSuite) TestACIPingGateway(c *C) {
-	if s.fwdMode == "routing" {
+	if s.fwdMode == "routing" || s.basicInfo.Scheduler == "k8" {
```

Comment:
the reason k8 check was added was because the code was not in sync with the new framework. this can be removed now.